Content.Shared/DeltaV/Addictions/AddictedComponent.cs

```diff
@@ -0,0 +1,21 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Timing;
+
+namespace Content.Shared.DeltaV.Addictions;
+
+[RegisterComponent, NetworkedComponent]
+public sealed partial class AddictedComponent : Component
+{
+    /// <summary>
+    ///     Whether to suppress pop-ups.
+    /// </summary>
+    [DataField]
```

Comment:
add AutoNetworkedField

Reply:
if we use AutoNetworkedField wouldn't client state always be replicated to the server? that's not what we want, since it's only ever modified by the UpdateSupressed() function on the server code which also always marks it as dirty

Reply:
dirtying a component with no state is useless, dirty tells server to send new state to all clients in pvs range